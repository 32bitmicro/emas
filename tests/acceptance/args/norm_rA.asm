lw r0, r0
lw r1, r0
lw r2, r0
lw r3, r0
lw r4, r0
lw r5, r0
lw r6, r0
lw r7, r0
