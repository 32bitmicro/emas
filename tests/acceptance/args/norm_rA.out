@ 0x0000 : 0x4000  /  010 000 0 000 000 000  /  16384
@ 0x0001 : 0x4040  /  010 000 0 001 000 000  /  16448
@ 0x0002 : 0x4080  /  010 000 0 010 000 000  /  16512
@ 0x0003 : 0x40c0  /  010 000 0 011 000 000  /  16576
@ 0x0004 : 0x4100  /  010 000 0 100 000 000  /  16640
@ 0x0005 : 0x4140  /  010 000 0 101 000 000  /  16704
@ 0x0006 : 0x4180  /  010 000 0 110 000 000  /  16768
@ 0x0007 : 0x41c0  /  010 000 0 111 000 000  /  16832
